frontend/app/locales/en/translations.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  'tasks.index.title': 'View tasks',
```

Comment:
yes! nice catch... we should be using nested objects. I thought that it didn't support it, but turns out it does. I shall change